commit 7a70a5dea2009cb6387182993953944290b1148a
Author: popcornmix <popcornmix@gmail.com>
Date:   Wed Dec 18 22:16:19 2013 +0000

    config: Enable CONFIG_MEMCG, but leave it disabled (due to memory cost). Enable with cgroup_enable=memory.

diff --git a/kernel/cgroup.c b/kernel/cgroup.c
index 85bc9beb046d..4acdbef46a8f 100644
--- a/kernel/cgroup.c
+++ b/kernel/cgroup.c
@@ -5629,7 +5629,7 @@ int __init cgroup_init_early(void)
 	return 0;
 }
 
-static u16 cgroup_disable_mask __initdata;
+static u16 cgroup_disable_mask __initdata = 1<<0;
 
 /**
  * cgroup_init - cgroup initialization
@@ -6166,6 +6166,28 @@ static int __init cgroup_no_v1(char *str)
 }
 __setup("cgroup_no_v1=", cgroup_no_v1);
 
+static int __init cgroup_enable(char *str)
+{
+	struct cgroup_subsys *ss;
+	char *token;
+	int i;
+
+	while ((token = strsep(&str, ",")) != NULL) {
+		if (!*token)
+			continue;
+
+		for_each_subsys(ss, i) {
+			if (strcmp(token, ss->name) &&
+			    strcmp(token, ss->legacy_name))
+				continue;
+
+			cgroup_disable_mask &= ~(1 << i);
+		}
+	}
+	return 1;
+}
+__setup("cgroup_enable=", cgroup_enable);
+
 /**
  * css_tryget_online_from_dir - get corresponding css from a cgroup dentry
  * @dentry: directory dentry of interest
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1059_7a70a5dea2009cb6387182993953944290b1148a.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1059_7a70a5dea2009cb6387182993953944290b1148a.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
